<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRecord</class>
 <widget class="QDialog" name="addRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ekle</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>YENİ ÖĞRENCİ</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="OgrenciLabel">
     <property name="text">
      <string>Öğrenci</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="OgrenciComboBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="bookLabel">
     <property name="text">
      <string>Aldığı Kitap</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="bookLineEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDateTimeEdit" name="deliveryDateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="deliveryDateLabel">
     <property name="text">
      <string>Veriliş Tarihi</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="sayfaSayisiLineEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="sayfaSayisiLabel">
     <property name="text">
      <string>Sayfa Sayısı</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="sonIadeTarihiLabel">
     <property name="text">
      <string>Son İade Tarihi</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDateTimeEdit" name="sonIadeTarihiDateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>OgrenciComboBox</tabstop>
  <tabstop>bookLineEdit</tabstop>
  <tabstop>sayfaSayisiLineEdit</tabstop>
  <tabstop>deliveryDateEdit</tabstop>
  <tabstop>sonIadeTarihiDateTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addRecord</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addRecord</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
